src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -260,6 +267,102 @@ public bool Read()
             return retVal;
         }
 
+        /// <summary>
+        /// Skips the children of the current JSON token.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown when the reader was given partial data with more data to follow (i.e. _isFinalBlock is false).
+        /// </exception>
+        public void Skip()
```

Comment:
Consider adding something like
```C#
/// <remarks>
/// When <see cref="TokenType"/> is <see cref="JsonTokenType.PropertyName" /> the reader first moves to the property value.
/// When <see cref="TokenType"/> (originally, or after advancing) is <see cref="JsonTokenType.StartObject" /> or 
/// <see cref="JsonTokenType.StartArray" /> the reader advances to the matching
/// <see cref="JsonTokenType.EndObject" /> or <see cref="JsonTokenType.EndArray" />.
///
/// For all other token types the reader does not move. After the next call to <see cref="Read"/> the reader will be at
/// the next value (when in an array) or the next property name (when in an object).
/// </remarks>

Reply:
Funnily, I had considered adding this :)